CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 **Bug Fixes**
 - Invalidate attribute cache entry on `PathAlreadyExists` error in create directory operation.
 
+**Features**
+- Support CPK for block storage accounts.
```

Comment:
```suggestion
- Support CPK encryption for blob storage accounts.
```

Reply:
We can keep this as `block`, since in our config we mention either `block` or `adls` storage accounts.